docs/alerts/understanding-alerts/what-is-a-netdata-alert.md

```diff
@@ -0,0 +1,99 @@
+# 1.1 What is a Netdata Alert?
+
+A **Netdata alert** is a rule that continuously monitors one or more metrics from charts and assigns a **status** based on whether configured conditions are met. Alerts are the core of Netdata's health monitoring system, acting as component-level watchdogs that evaluate metrics at regular intervals.
+
+## How Alerts Work
+
+At each evaluation cycle (typically every 10 seconds, configurable per alert via the `every` line):
+
+1. The **health engine** running on the Agent or Parent reads recent metric data from the relevant chart(s)
+2. It applies the alert's logic (for example, "average CPU over the last 5 minutes > 80%")
+3. It assigns a **status** to that alert based on the result
+4. If the status changes (for example, from `CLEAR` to `WARNING`), this transition becomes an **alert event**
+
+Alert events are:
+- Visible in the local Agent dashboard
+- Queryable via Agent APIs (`/api/v1/alarms`, `/api/v1/alarm_log`)
+- Sent to Netdata Cloud and displayed in the Events Tab
+- Used to trigger notifications (email, Slack, PagerDuty, etc.)
+
+## Alert Statuses
+
+Every alert has one of six possible statuses at any given time:
+
+| Status | Meaning |
+|--------|---------|
+| **UNINITIALIZED** | The alert has just been created but has not yet collected enough data to evaluate |
+| **UNDEFINED** | The alert could not be evaluated (missing data, division by zero, NaN/Inf result) |
+| **CLEAR** | The alert can be evaluated and neither warning nor critical thresholds are triggered; the metric is in a normal, healthy state |
+| **WARNING** | The warning expression evaluated to true; something may require attention |
+| **CRITICAL** | The critical expression evaluated to true; if both warning and critical conditions are met, CRITICAL takes precedence |
+| **REMOVED** | The alert has been deleted (configuration reload, child node disconnected, or agent shutdown) |
+
+### Status Lifecycle
+
+Alerts typically flow through this lifecycle:
+
+```
+UNINITIALIZED → UNDEFINED / CLEAR → WARNING → CRITICAL
+```
+
+- New alerts start in `UNINITIALIZED` until enough data is collected
+- Once data is available, they move to `CLEAR` (healthy) or `UNDEFINED` (cannot evaluate)
+- From `CLEAR`, they can transition to `WARNING` or `CRITICAL` when conditions are met
+- They return to `CLEAR` when conditions are no longer met
+- `REMOVED` appears when an alert is deleted during configuration reload or agent shutdown
```

Comment:
Basically when an associated chart instance is no longer getting new data (obsolete). Configuration reload is a subset of this, but it could also be because a device or service that is no longer there. Agent shutdown itself does not cause REMOVED alerts.